src/components/pages/offer/__specs__/Offer.spec.jsx

```diff
@@ -0,0 +1,73 @@
+import { createMemoryHistory } from 'history'
+import { mount } from 'enzyme'
+import { Provider } from 'react-redux'
+import React from 'react'
+import { Router } from 'react-router'
+
+import AbsoluteFooterContainer from '../../../layout/AbsoluteFooter/AbsoluteFooterContainer'
+import DetailsContainer from '../../../layout/Details/DetailsContainer'
+import HeaderContainer from '../../../layout/Header/HeaderContainer'
+import OfferContainer from '../OfferContainer'
+import getMockStore from '../../../../utils/mockStore'
+
+jest.mock('../../../hocs/with-login/withRequiredLogin', () => WrappedComponent => props => (
+  <WrappedComponent {...props} />
+))
+
+describe('offerContainer', () => {
+  let props
+
+  beforeEach(() => {
+    props = {
+      match: {
+        params: {
+          details: 'details',
+          offerId: 'o1'
+        }
+      }
+    }
+  })
+
+  describe('when I am log-in', () => {
```

Comment:
```suggestion
  describe('when I am logged in', () => {
```
Je ne vois pas le setup correspondant au describe